### PR TITLE
Try fixing templates test.

### LINKED ISSUE #3157: Story Templates test failing.
The test was temporarily skipped in https://github.com/ampproject/amp-wp/pull/3156 but needs to be investigated and fixed.

### PR DESCRIPTION
**Pull request by miina**
Fixes #3157.

### CODE REVIEW

**Comment by miina** [comment]
Passed the tests 3 times 🤞

File: tests/e2e/specs/stories-editor/story-templates.js [hunk 1]
```diff
@@ -72,9 +73,7 @@ describe( 'Story Templates', () => {
 				await page.waitForNavigation();
 			} );
 
-			// @todo Figure out why it's failing.
-			// Reason for skipping: the test is sometimes failing, will need to investigate this separately.
-			it.skip( 'should display non-template reusable blocks in the reusable blocks management screen', async () => { // eslint-disable-line jest/no-disabled-tests
+			it( 'should display non-template reusable blocks in the reusable blocks management screen', async () => { // eslint-disable-line jest/no-disabled-tests
```
Comment (swissspidy) [inline tests/e2e/specs/stories-editor/story-templates.js]:
```suggestion
			it( 'should display non-template reusable blocks in the reusable blocks management screen', async () => {
```

Comment (swissspidy) [reply tests/e2e/specs/stories-editor/story-templates.js]: No idea why `eslint-comments/no-unused-disable` is not catching this though 🤔

**Comment by miina** [comment]
Looks like the suggested changes work :)

**Pull request merged**